unlock-js/src/__tests__/helpers/nockHelper.js

```diff
@@ -127,7 +127,11 @@ export class NockHelper {
     this.anyRequestSetUp = true // detect http calls made before any mocks setup
     const cb = (...args) => this.logNock(args)
     return this.nockScope
-      .post('/', { jsonrpc: '2.0', id: this._rpcRequestId, method, params })
+      .post('/', body => {
+        // This is a matcher function, the nock scope is limited only to calls
+        // that have this jsonrpc version and the exact method being called.
+        return body.jsonrpc === '2.0' && body.method === method
```

Comment:
Very good! I actually we could eventually leverage that to add just the stuff we need to check.